api/models/Group.js

```diff
@@ -471,7 +472,7 @@ module.exports = bookshelf.Model.extend(merge({
           // Make sure that the user making the changes doesn't need to then accept the new agreements
           const updatedByUserMembership = await GroupMembership.forPair(updatedByUserId, this.id).fetch()
           if (updatedByUserMembership) {
-            await updatedByUserMembership.save({ settings: { agreementsAcceptedAt: (new Date()).toISOString() } }, { transacting })
+            await updatedByUserMembership.save({ settings: Object.assign(updatedByUserMembership.settings, { agreementsAcceptedAt: (new Date()).toISOString() }) }, { transacting })
```

Comment:
Any particular reason to use Object.assign instead of the spread operator here?

Reply:
in the moment i wasnt getting spread to work there 🤷‍♂️